libfuse/fs.go

```diff
@@ -79,6 +111,84 @@ func NewFS(config libkbfs.Config, conn *fuse.Conn, debug bool, platformParams Pl
 	return fs
 }
 
+// tcpKeepAliveListener is copied from net/http/server.go, since it is
+// used in http.(*Server).ListenAndServe() which we want to emulate in
+// enableDebugServer.
+type tcpKeepAliveListener struct {
+	*net.TCPListener
+}
+
+func (ln tcpKeepAliveListener) Accept() (c net.Conn, err error) {
```

Comment:
What is `ln`?  "listener", maybe?  Can we change to `tkal` or something to be consistent with using acronyms elsewhere?